tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Suggest expanding this text block to define and link to terms needed to understand the following code block, which contains acronyms that may not be familiar to readers.</p><ul><li>OLI</li><li>TM</li><li>ETM+</li><li>CFMask</li><li>NBR</li><li>understanding Earth Engine joins</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='98'/>

Reply:
<p>Done.</p>